File: src/main/java/team4/footwithme/vote/repository/VoteItemRepository.java
package team4.footwithme.vote.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import team4.footwithme.vote.domain.VoteItem;
import team4.footwithme.vote.domain.VoteItemLocate;

import java.util.List;

@Repository
public interface VoteItemRepository extends JpaRepository<VoteItem, Long> {

    List<VoteItem> findByVoteVoteId(Long voteId);
}

File: src/main/java/team4/footwithme/vote/repository/VoteItemRepository.kt
package team4.footwithme.vote.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import team4.footwithme.vote.domain.VoteItem

@Repository
interface VoteItemRepository : JpaRepository<VoteItem?, Long?> {
    fun findByVoteVoteId(voteId: Long?): List<VoteItem?>?
}
